src/wp-includes/functions.php

```diff
@@ -8860,6 +8860,39 @@ function wp_fuzzy_number_match( $expected, $actual, $precision = 1 ) {
 	return abs( (float) $expected - (float) $actual ) <= $precision;
 }
 
+
+/**
+ * Replaces trailing period, if any, with encoded entity in URLs.
+ *
+ * If the URL ends with a period, replace it with %2E to avoid
+ * misinterpretation from some email clients.
+ *
+ * @since 5.9.0
+ *
+ * @param string $url The URL to encode.
+ * @return string The new encoded URL.
+ */
+function _encode_query_arg_ending_period( $url ) {
```

Comment:
Does this function private?

Reply:
I think I found out the way to do it is @access private? I've made a change to do that now